### PR TITLE
First version homepage

### PR DESCRIPTION
**Pull request by Ksprbgh**
Doing this because the strange man asks me to...

@Jeannettee,

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,11 @@
...
+**Seriøst**
+
+Nu bliver det for alvor alvorligt
+
+Here is the [Help-file](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+
+
+
+There are emojis on here :+1:
```
Comment (bitoiu) [inline README.md:8]: Why so many whitespaces in between the two sentences.

File: README.md [hunk 2]
```diff
@@ -1 +1,11 @@
 # ksprbgh.github.io
+**Seriøst**
+
+Nu bliver det for alvor alvorligt
```
Comment (bitoiu) [inline README.md]: can't understand it :cry:

File: README.md (hunk 1 above)
Comment (kkh9000) [inline README.md:8]: Consider reversing.

Comment (xmacex) [inline README.md:8]: Good question @bitoiu, since space is black

**Comment by jbthestrup** [comment]
Should we not have a link to Twitter?

**Pull request merged**